etc/templates/ports_test.json

```diff
@@ -1,8 +1,10 @@
-{
-  "Top Module": "NAME_OF_THE_TOP_MODULE",
-  "Ports": [
-    {"name": "inp1", "direction": "Input", "range": {"msb":0, "lsb":0}, "type": "LOGIC"},
-    {"name": "out1", "direction": "Output", "range": {"msb":4, "lsb":0}, "type": "REG"},
-    {"name": "inout1", "direction": "Inout", "range": {"msb":2, "lsb":0}, "type": "LOGIC"}
-  ]
-}
+[
```

Comment:
rename the file to ports_info.json, same as the upstream one for 3rd party users.

Reply:
done